src/main/java/org/jenkinsci/plugins/corsfilter/AccessControlsFilter.java

```diff
@@ -34,7 +34,6 @@
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
     private static final Logger LOGGER = Logger.getLogger(AccessControlsFilter.class.getCanonicalName());
     private static final String PREFLIGHT_REQUEST = "OPTIONS";
-    private List<String> allowedOriginsList = null;
```

Comment:
:bug: I think this field should be marked as `transient` instead of being removed. This will most likely cause xstream issues when trying to load this object.

Reply:
This is not part of the configuration but a list created the first time the isAllowed method is executed. No xstream issue.

Reply:
This object is not persisted, so it should be fine. The code is not ideal, but this removal should be safe